assets/email/failed-login-attempt.html

```diff
@@ -0,0 +1,275 @@
+                            <!-- START MAIN CONTENT AREA -->
+                            <tr>
+                                <td
+                                    class="wrapper"
+                                    style="
+                                        font-family: sans-serif;
+                                        font-size: 14px;
+                                        vertical-align: top;
+                                        box-sizing: border-box;
+                                        padding: 20px;
+                                    "
+                                >
+                                    <table
+                                        border="0"
+                                        cellpadding="0"
+                                        cellspacing="0"
+                                        style="
+                                            border-collapse: separate;
+                                            mso-table-lspace: 0pt;
+                                            mso-table-rspace: 0pt;
+                                            width: 100%;
+                                        "
+                                    >
+                                        <tr>
+                                            <td style="font-family: sans-serif; font-size: 14px; vertical-align: top">
+                                                <p
+                                                    style="
+                                                        font-family: sans-serif;
+                                                        font-size: 14px;
+                                                        font-weight: normal;
+                                                        margin: 0;
+                                                        margin-bottom: 15px;
+                                                    "
+                                                >
+                                                    Hi there!
+                                                </p>
+
+                                                <p
+                                                    style="
+                                                        font-family: sans-serif;
+                                                        font-size: 14px;
+                                                        font-weight: normal;
+                                                        margin: 0;
+                                                        margin-bottom: 15px;
+                                                    "
+                                                >
+                                                    This is just an email to warn you that there was a
+                                                    <strong>failed</strong> attempt to login to your account in Padloc,
+                                                    from {{ location }}.
```

Comment:
The variable's called `location` but it _can_ include the device info as well. I just didn't want to name it `locationAndPotentiallyDeviceToo`.